extension/oidcauthextension/metadata.yaml

```diff
@@ -7,3 +7,7 @@ status:
   distributions: [contrib, observiq, sumo]
   codeowners:
     active: [jpkrohling]
+
+tests:
+  config:
+  skip_lifecycle: true
```

Comment:
misaligned

Reply:
`skip_lifecycle` is a flag outside of config to change the way we generate the test.

Reply:
Oh, right. My bad